### PR TITLE
Fix/warning text disclaimer link title

### PR DESCRIPTION
**Pull request by chrisrhymes**
Update the WarningTextDisclaimer link text and [remove the title](https://www.a11yproject.com/posts/creating-valid-and-accessible-links/#what-about-the-title-attribute%3F). 

Added test for the parent WarningText component and updated the files with prettier for consistency.

### CODE REVIEW

File: src/library/slices/WarningTextDisclaimer/WarningTextDisclaimer.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+import React from 'react';
+import { WarningTextDisclaimerProps } from './WarningTextDisclaimer.types';
+import WarningText from '../WarningText/WarningText';
+
+const WarningTextDisclaimer: React.FunctionComponent<WarningTextDisclaimerProps> = () => {
+  return (
+    <WarningText title="Why information is on a different website" isWarning>
+      We are in the process of adding information to this new unitary council website. Some pages will give you a link
+      back to a previous council website to help you find what you need.{' '}
+      <a href={'/your-council/about-website'}>Read more about the council changes</a>.
```
Comment (chrisrhymes) [inline src/library/slices/WarningTextDisclaimer/WarningTextDisclaimer.tsx:10]: The actual change is removing the title and updating the link text

File: src/library/slices/WarningText/WarningText.tsx [hunk 2]
```diff
@@ -1,17 +1,18 @@
+import React from 'react';
+import parse from 'html-react-parser';
+import { WarningTextProps } from './WarningText.types';
+import * as Styles from './WarningText.styles';
 
-import React from "react";
-import parse from "html-react-parser"
+// @TODO this needs to fit within the content border
```
Comment (DamienBurkeWN) [inline src/library/slices/WarningText/WarningText.tsx:6]: Is this comment still relevant?

**Review by DamienBurkeWN — CHANGES_REQUESTED** [review]
Is the TODO still to be actioned or ...?

File: src/library/slices/WarningText/WarningText.tsx (hunk 2 above)
Comment (chrisrhymes) [reply src/library/slices/WarningText/WarningText.tsx:6]: I was going to ask you the same thing. Do you have any idea what it refers to?

Comment (DamienBurkeWN) [reply src/library/slices/WarningText/WarningText.tsx:6]: Nope!

**Review by DamienBurkeWN — APPROVED** [review]
👍

**Pull request merged**